src/telemetry/Counter.cc

```diff
@@ -21,6 +23,8 @@ double Counter::Value() const noexcept {
     return handle.Value();
 }
 
+void Counter::ClearHandle() { family->Remove(&handle); }
```

Comment:
Wasn't able to find the user of `ClearHandle()`, so would suggest to remove it. It's existence might be an incentive to link less about labels and whether unbounded metrics are produced, which we don't want.

Also for Gauge.